test/app-luatest/tnt_debug_getsources_test.lua

```diff
@@ -0,0 +1,45 @@
+local t = require('luatest')
+local fio = require('fio')
+local g = t.group()
+
+
+local function readfile(filename)
+    local f = assert(io.open(filename, "rb"))
+    return f:read('*a')
+end
+
+local files = {
+    'strict',
+    'debug',
+    'errno',
+    'fiber',
+    'env',
+    'datetime',
+}
+
+g.test_tarantool_debug_getsources = function()
+
+    local fh = io.popen('git rev-parse --show-toplevel', 'r')
```

Comment:
Too complex to me. It's better to calculate relative path from the test source file.

Reply:
I've reimplemented it using `debug.getinfo(1,'S')`

```diff
diff --git a/test/app-luatest/tnt_debug_getsources_test.lua b/test/app-luatest/tnt_debug_getsources_test.lua
index 3749fba03..3658a6f89 100644
--- a/test/app-luatest/tnt_debug_getsources_test.lua
+++ b/test/app-luatest/tnt_debug_getsources_test.lua
@@ -17,11 +17,16 @@ local files = {
     'datetime',
 }
 
-g.test_tarantool_debug_getsources = function()
+-- calculate reporsitory root using directory of a current
+-- script, but get 2 directories above
+local function repo_root()
+    local myself = fio.abspath(debug.getinfo(1,'S').source:gsub("^@", ""))
+    local root = fio.abspath(fio.dirname(myself) .. '/../..')
+    return root
+end
 
-    local fh = io.popen('git rev-parse --show-toplevel', 'r')
-    local git_root = fh:read('*l')
-    fh:close()
+g.test_tarantool_debug_getsources = function()
+    local git_root = repo_root()
     t.assert_is_not(git_root, nil)
     t.assert(fio.stat(git_root):is_dir())
     local lua_src_dir = fio.pathjoin(git_root, '/src/lua')
diff --git a/third_party/c-dt b/third_party/c-dt
```